### DISCUSSION TITLE
websocket连接错误

### DISCUSSION THREAD

**Discussion by XingShuyu** [Tsuk1ko/CQ-picfinder-robot#229] [问题求助] [answered]
我第一次启动cq时，反复提示连接错误。
![Screenshot_20210724_090533](https://user-images.githubusercontent.com/82715884/126853219-d0fe6324-d8cc-48f1-81c1-1c5ca11f69d7.jpg)

我检查了go-cqhttp的配置，cq的配置都没问题（截图
![Screenshot_20210724_090234](https://user-images.githubusercontent.com/82715884/126853199-1c00053a-9514-485d-8c34-7aa9cf7f4a42.jpg)
![Screenshot_20210724_090341](https://user-images.githubusercontent.com/82715884/126853202-8c2dc345-a9f7-467f-89a0-9b2a51f80d79.jpg)
），部署过程也没有错误。谁能帮忙解决这个问题？我该怎么办？

**Comment by Miuzarte** [comment, marked as answer]
gocq有开着吗(

**Reply by XingShuyu** [reply]
开着，我怀疑是我用Android proot的缘故

**Reply by XingShuyu** [reply]
termux的proot虚拟机

**Reply by Miuzarte** [reply]
> 开着，我怀疑是我用Android proot的缘故

那玩意我就没yarn成功过，挺迷的

**Reply by XingShuyu** [reply]
确实，我都是用npm i
